master/pyext/src/validation/__init__.py

```diff
@@ -16,6 +16,42 @@
 from collections import defaultdict
 from validation import utility
 
+import logging
+from typing import Final
+
+#########################
+# Setup operational mode
+#########################
+
+logging.basicConfig(level=logging.INFO)
+IHMV_MODES: Final = ["PRODUCTION", "DEVELOPMENT"]  # Available modes
```

Comment:
This makes a global variable which litters the namespace. Consider putting it inside the `get_operational_mode` function instead.

Reply:
I wonder if replace it with enum would it be ok to keep it in the root namespace? Something like:

```
class IHMVAvailableModes(Enum):
    PRODUCTION = 0
    DEVELOPMENT = 1

```

and then checks like:

```

# Alter variables for the DEVELOPMENT mode
if IHMV_MODE.name == IHMVAvailableModes.DEVELOPMENT:
    __max_num_models = 20  # Cap number of structures for development purposes
```

Reply:
yeah, I'll update it today.